pkg/cloud/gcp/gcs_storage.go

```diff
@@ -45,6 +46,9 @@ const (
 	CredentialsParam = "CREDENTIALS"
 	// AssumeRoleParam is the email of the service account to assume.
 	AssumeRoleParam = "ASSUME_ROLE"
+	// TemporaryTokenParam is the query parameter for a temporary credentials
+	// token.
+	TemporaryTokenParam = "TEMPORARY_TOKEN"
```

Comment:
Happy to defer to others here, but I'm not sure `TEMPORARY_TOKEN` is the clearest name. Perhaps I'm just unfamiliar with all the terminology, but I don't think `TEMPORARY_TOKEN` maps to terminology in either GCP or OAuth2.  Perhaps something like `STATIC_ACCESS_TOKEN` would be better.  "Access Token" is mentioned in the gcp docs and is a well known OAuth2 term.

Reply:
Using `BEARER_TOKEN` instead per @catj-cockroach 's comment.